Add SD card content to SDBlockDevice.md

AnotherButler opened the pull request:
Add content about wiring an SD card module to a developer board.

Commit:
Add SD card content to SDBlockDevice.md

Add content about wiring an SD card module to a developer board.

Commit:
Fix image link in SDBlockDevice.md

Fix image link to have correct depth.

Commit:
Update image link in SDBlockDevice.md

Update image link to point to renamed image.

teetak01 commented on docs/api/storage/SDBlockDevice.md:
```diff
@@ -6,6 +6,17 @@ You can use the Mbed OS SD card block device, so applications can read and write
 
+## How to wire an external SD-card module to a developer board
+
+<span class="images">![](../images/sd-wiring.png)<span>How to wire an external SD-card module to a developer board</span></span>
```
Should this PR also include the .png or how does that work?

AnotherButler replied:
I added the PR separately: https://github.com/ARMmbed/mbed-os-5-docs/tree/development/docs/images

teetak01 replied:
Ok, a bit unclear why you would do those in seperate PRs.

teetak01 reviewed (commented):
Should this also mention that application needs to override the SD pin mapping based on the wiring?

For. ex. as is done in https://github.com/ARMmbed/mbed-cloud-client-example/blob/master/mbed_app.json#L123-L126

VeijoPesonen commented:
Not to forget that the "SD" component needs to be added - [Component SD](https://github.com/ARMmbed/mbed-cloud-client-example/blob/fd495ae5d9c94e3501cc17bc2a71d51ed1487179/mbed_app.json#L109)

Commit:
Add content to SDBlockDevice.md

Add content requested in comments about adding the SD component and overriding the pin mapping.

AnotherButler commented:
@VeijoPesonen and @teetak01 Do these changes look correct?

Commit:
Update image in SDBlockDevice.md

Update image link.
